src/Microsoft.Framework.ConfigurationModel.Json/Resources.resx

```diff
@@ -123,6 +123,9 @@
   <data name="Error_CommitWhenNewKeyFound" xml:space="preserve">
     <value>Unable to commit because a new key was added to the configuration file after last load operation. The newly added key is '{0}'.</value>
   </data>
+  <data name="Error_FileNotFound" xml:space="preserve">
+    <value>The file '{0}' was not found and is not set optional.</value>
```

Comment:
Change to: `The configuration file '{0}' was not found and is not optional.` (in all the resx files)